### ISSUE TITLE
[Bug]: Wrong gap between the video title and view count

### ISSUE THREAD

**Issue by PanMarSagYT** [sapondanaisriwan/AdashimaaTube#99]
### Describe the bug

I noticed that the gap between video title and view count is 10 pixels larger.

### Screenshots

![chrome_VAMvaixeSS](https://github.com/sapondanaisriwan/AdashimaaTube/assets/81510862/ccf215a7-200c-483d-b4e4-49ce0e1a2e2b)
It should look like this:
![correct spacing](https://github.com/sapondanaisriwan/AdashimaaTube/assets/81510862/69303d0e-757f-49a1-b351-e9addd30cd1c)

### What browser are you using?

Chrome version 113.0.0.0 (64-bit)

### What version of AdashimaaTube are you using?

v2.2.6

**Comment by sapondanaisriwan** [comment]
Yep I also noticed that but it doesn't ruin your watch experience. I'm going to fix that until RYD fix the old description panel.

**Comment by Rockerfelle** [comment]
I'm still having this spacing problem- has a fix been released yet?
